tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -0,0 +1,164 @@
+gcloud services enable cloudapis.googleapis.com 
+gcloud services enable compute.googleapis.com
+gcloud services enable servicemanagement.googleapis.com 
+gcloud services enable storage-api.googleapis.com  
+gcloud services enable cloudbuild.googleapis.com
+```
+
+## Task 3 Give the Cloud Build user Project Editor permissions 
+```sh
+CLOUD_BUILD_ACCOUNT=$(gcloud projects get-iam-policy $PROJECT --filter="(bindings.role:roles/cloudbuild)"  --flatten="bindings[].members" --format="value(bindings.members[])")
+
+gcloud projects add-iam-policy-binding $PROJECT \
+  --member $CLOUD_BUILD_ACCOUNT \
+  --role roles/editor   
+```
+
+## Task 4 Create the Cloud Source Repository for your image creator 
+```sh
+gcloud source repos create helloworld-image-factory
+```
+
+## Task 5 Create the build triggers for the image creator source repository 
+Go to to the [build triggers page](https://console.cloud.google.com/cloud-build/triggers) and create a trigger.  
+
+1. Click "Create Trigger" 
+1. Select "Cloud Source Repository" Click "Continue".
+1. Select "helloworld-image-factory" anc click "Continue" 
+1. Enter "Hello world image factory" for Name." 
+1. Set the trigger for "Tag". 
+1. Set the build type to "cloudbuild.yaml"
+1. Set the substitution, ``_IMAGE_FAMILY`` to centos-7
+1. Set the substitution, ``_IMAGE_ZONE`` to the zone you want to use the value of ```$ZONE```. 
+1. Click "Create Trigger" 
+
+**Note: To see a list of image families:** 
+```sh
+gcloud compute images list | awk '{print $3}'  | awk '!a[$0]++'
+```
+
+## Task 6 Add the packer Cloud Build image to your project
+
+```sh
+project_dir=$(pwd)
+cd /tmp
+git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
+cd cloud-builders-community/packer
+gcloud builds submit --config cloudbuild.yaml
+rm -rf /tmp/cloud-builders-community
+cd $project_dir
+```
+
+## Task 7 Add your helloworld-image-factory google repository as a remote repository with the name 'google'
+1. (Only if not running in Cloud Shell) setup your google credentials for git. 
+```sh 
+gcloud init && git config --global credential.https://source.developers.google.com.helper gcloud.sh
+```
+2. Add the google cloud repo as a remote. 
+```sh 
+git remote add google \
+  https://source.developers.google.com/p/$PROJECT/r/helloworld-image-factory
+```
+
+## Task 8 Push the repository and tags to google
+1. Tag the repository with a version number.
+```sh 
+git tag v0.1
+```
+2. Push the branch and the tags to your google repository.
+```sh
+git push google master --tags
+```
+
+## Task 9 View build progress 
+1. Open up the [Cloud Build](https://console.cloud.google.com/cloud-build) console to show the build progress.
+2. Find the build that is in progress and click the link to view its progress. 
+
+## Task 10 Create a compute instance for the image in your gcp project 
+1. Once the build completes, create the instance and requisite firewall rule to test that the image works.
```

Comment:
If there's only one step in a heading, don't put it in a list.

Reply:
Fixed.